starter_code/src/movies.js

```diff
@@ -73,6 +73,10 @@ function ratesAverage (arr) {
 	
 	const ratesArr = [];
 
+	if (arr = []) {
+		return 0;
+	}
+
```

Comment:
to check if an array is empty, you want to check if length of array is equal to 0
if (arr.length === 0) 
or 
if (!arr.length)